AxonDeepSeg/ads_utils.py

```diff
@@ -282,4 +282,4 @@ def get_existing_models_list():
     return models_list
 
 # Call init_ads() automatically when module is imported
-init_ads()
+# init_ads()
```

Comment:
This line needs to be uncommented I think, as the error tracking depends on it. If doing so causes a problem, then we should open an issue about it.

Reply:
Unable to resolve quickly - opened issue #285 about it.